OpenEdXMobile/res/values/styles.xml

```diff
@@ -582,15 +582,24 @@
         <item name="lineHeight">@dimen/widget_margin_double</item>
     </style>
 
-    <style name="EmptyStateButton" parent="edX.Widget.ActionBrandPillButton">
+    <style name="CourseDashboardHeadline">
+        <item name="android:textColor">@color/neutralBlackT</item>
+        <item name="android:textSize">@dimen/edx_large</item>
+        <item name="android:gravity">start</item>
+        <item name="lineHeight">@dimen/widget_margin_x_double</item>
+    </style>
+
+    <style name="PrimaryActionButton">
+        <item name="android:layout_width">match_parent</item>
         <item name="android:layout_height">wrap_content</item>
-        <item name="android:layout_margin">@dimen/widget_margin_x_double</item>
-        <item name="android:text">@string/find_course_btn_text</item>
-        <item name="android:textSize">@dimen/edx_x_large</item>
-        <item name="backgroundTint">@color/secondaryBaseColor</item>
-        <item name="icon">@drawable/ic_search</item>
-        <item name="iconGravity">textStart</item>
-        <item name="iconPadding">@dimen/container_padding</item>
+        <item name="android:minHeight">@dimen/edx_button_height</item>
+        <item name="android:layout_marginTop">@dimen/edx_default_margin_double</item>
+        <item name="android:background">@drawable/edx_brand_button</item>
+        <item name="backgroundTint">@null</item>
+        <item name="android:textColor">@color/neutralWhite</item>
+        <item name="android:textSize">@dimen/edx_small</item>
```

Comment:
IMO `edx_base` text is more appropriate here.

Reply:
As per sigma it is `edx_small`

Reply:
Figma file shows dimens in `px`, and in Android, we are using `dp`, and `sp`, which are not equal to the `px` after conversion.